src/components/collapsible-motion/collapsible-motion.js

```diff
@@ -152,8 +153,6 @@ const UncontrolledCollapsibleMotion = props => {
           };
         }
 
-        console.log('here', animation);
```

Comment:
Yesterday I used logpoints in the Chrome DevTools for the first time. Can only recoommend them for this kind of stuff.